manifests/v1beta1/installs/katib-standalone-openshift/kustomization.yml

```diff
@@ -0,0 +1,87 @@
+# This kustomization copies the `katib-standalone` one with following exclusions:
```

Comment:
Let's name this install: `katib-openshift` to be consistent with `katib-ibm`.
What do you think @maanur  ?

Reply:
Sounds good